projects/core/src/i18n/translation-chunk.service.spec.ts

```diff
@@ -36,11 +36,33 @@ describe('TranslationChunkService', () => {
     });
 
     it('should return the key if no chunk was configured for it', () => {
-      expect(service.getChunkNameForKey('key2')).toBe('key2');
+      expect(service.getChunkNameForKey('key3')).toBe('key3');
     });
 
     it('should return the main part of the key if no chunk was configured for it', () => {
-      expect(service.getChunkNameForKey('key2.subKey2')).toBe('key2');
+      expect(service.getChunkNameForKey('key3.subKey2')).toBe('key3');
+    });
+  });
+
+  describe('I18n config', () => {
+    it('should composed of object of arrays', () => {
```

Comment:
Do we need this test? Isn't it ensured by the interface of the i18n chunks config?